packages/commonwealth/client/scripts/views/pages/CreateCommunity/CreateCommunity.tsx

```diff
@@ -14,6 +15,7 @@ import { CreateCommunityStep, getFormSteps } from './utils';
 import './CreateCommunity.scss';
 
 const CreateCommunity = () => {
+  const communityStakeEnabled = useFlag('flag.communityStake');
```

Comment:
is this typed somehow? if not, we should build some object/enum that will hold those values so we won't use strings all over the places